site/app/libraries/database/DatabaseQueries.php

```diff
@@ -178,6 +178,34 @@ public function getUsersByIds(array $user_id_list): ?array {
         return $this->getUsers($user_id_list);
     }
 
+    /**
+     * Gets a list of given and family names from the database given an array of IDs.
+     *
+     * @param string $u_or_t
+     * @param string $section_key
+     * @param string $users_or_teams
+     * @param string $user_or_team_id
+     * @param string $verified
+     * @param string $where
+     * @return string
+     */
+    public function getGrading($u_or_t, $section_key, $users_or_teams, $user_or_team_id, $verified, $where): ?string {
```

Comment:
This should probably also be a private function given that it should never be executed from outside this file.